tests/test_crud.py

```diff
@@ -0,0 +1,20 @@
+"""Test API to query subjects from the Stardog graph who match user-specified criteria."""
+
+
+def test_get_input(test_app):
+    """Given a valid sex string, returns a 200 status code."""
+    response = test_app.get("/query/?sex=female")
```

Comment:
All tests now mock the GET method aside from `test_app_with_invalid_environment_vars`, since the error handling for invalid USER/PASSWORD values happens inside the function in `crud.py` itself.